clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -153,15 +154,17 @@ public void testMetadataUpdateEvent() {
 
     @Test
     public void testAsyncCommitEvent() {
-        ApplicationEvent e = new AsyncCommitEvent(new HashMap<>());
+        Timer timer = time.timer(100);
```

Comment:
This should be `Long.MAX_VALUE` I think.

Reply:
OK.